cql/ManageCommonAbnormality.cql

```diff
@@ -482,7 +488,7 @@ define ErrataRecommendation:
     when ( // Errata Recommendation 3.1.1: ASC-US Alone
       AgeInYears() >= 25 and
       CytologyAloneInterpretedAsAscus and
-      Collate.MostRecentReport = MostRecentCytologyAlone.date and
+      Collate.HasNoBiopsyOrTreatmentAfterDate(MostRecentCytologyAlone.date) and
```

Comment:
This is a duplicate of the next line? Did you mean to remove or change the last line?

Reply:
That was supposed to be `IsMostRecentTest`. Good catch. I updated this in https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/135/commits/75868295f5ab1edce96613a67ff67244a483f303.